plover/gui_qt/main_window.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, engine, use_qt_notifications):
                 action.setObjectName(tool_plugin.name)
                 if tool.__doc__ is not None:
                     action.setToolTip(tool.__doc__)
-                if tool.SHORTCUT is not None:
+                if tool.SHORTCUT is not None and parent == tools_menu:
```

Comment:
Could you add a comment to the code to explain why this works? (both for me and for the next time someone ends up on this line of code)